app2/src/main/java/com/example/app2/service/OrderProcessingService.java

```diff
@@ -0,0 +1,68 @@
+package com.example.app2.service;
+
+import com.example.app1.pojo.Order;
+import com.example.app2.pojo.Summary;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.listener.adapter.JmsResponse;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderProcessingService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrderProcessingService.class);
+    private static final String SUMMARY_TOPIC = "orders.summary";
```

Comment:
would suggest to move it to application properties file:

`private static final String SUMMARY_TOPIC = "orders.summary";`

but it is up to you